src/_P023_OLED.ino

```diff
@@ -75,32 +76,24 @@ boolean Plugin_023(uint8_t function, struct EventStruct *event, String& string)
 
     case PLUGIN_WEBFORM_LOAD:
     {
-      addFormCheckBox(F("Use SH1106 controller"), F("p023_use_sh1106"), PCONFIG(5));
+      OLedFormController(F("p023_use_sh1106"), PCONFIG(5) == 1 ? 2 : 1); // Selector: 1 = SSD1306, 2 = SH1106
```

Comment:
Hmm this is highly non-intuitive.

Reply:
I have the intention to re-use the controller selector also used for P036 and P109, but have to convert the previously stored `0/1` bool state to the `1/2` state required by that selector, and back m(below). That does make the code look non-intuitive.
_Can_ revert to the previously used checkbox, if that is preferred.

Reply:
Well if you intend to make it right in the end, I will "see this one through the fingers" :)

Reply:
My intended improvement, the selector, is in place with this code, so I guess it will stay this 'awkward', unless a conversion on installed devices is to be added. But that would confuse anyone that would later revert to an older build, and also require a settings-version increase. Maybe a more elaborate comment could somewhat unconfuse things?

Reply:
Comments to unconfuse things is OK.

Reply:
Now it has been fixed much better.